.scripts/prune_registry.py

```
@@ -101,14 +101,14 @@ def get_packages_via_graphql():
             "org": ORG_NAME,
             "packageType": "DOCKER",
             "first": 100,
-            "after": cursor
+            "after": cursor,
         }
 
         try:
             res = requests.post(
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)